script/tool/test/mocks.dart

```diff
@@ -32,36 +33,49 @@ class MockPlatform extends Mock implements Platform {
 }
 
 class MockProcess extends Mock implements io.Process {
-  MockProcess();
-
-  /// A mock process that terminates with exitCode 0.
-  MockProcess.succeeding() {
```

Comment:
I added this a convenience recently because I didn't want to change its default behavior, but auditing `MockProcess` usage, as well as now needing to frequently pass stdout, it was clear that just using the default constructor for success was simpler.